src/chapters/intro-stats-data-modeling/intro-stats-data-modeling.rst

```diff
@@ -1,2 +1,141 @@
 Introduction to Statistics and Data Modeling
-============================================
\ No newline at end of file
+============================================
+
+Readings
+--------
+
+Read the following articles, and watch the videos.
+
+
+`Descriptive vs. Inferential Statistics <https://www.thoughtco.com/differences-in-descriptive-and-inferential-statistics-3126224>`__.
+
+* Don't be afraid to explore the links in the article to get a more in depth understanding of the material.
+
+`Understanding Descriptive Statistics | by Sarang Narkhede <https://towardsdatascience.com/understanding-descriptive-statistics-c9c2b0641291>`__.
+
+`How to choose the best chart or graph for your data <https://cloud.google.com/blog/products/data-analytics/different-types-graphs-charts-uses>`__.
+
+`Python Statistics Fundamentals: How to Describe Your Data – Real Python <https://realpython.com/python-statistics/#visualizing-data>`__.
+
+`Descriptive statistics and data visualization. An introduction to statistics and working with data <https://www.youtube.com/watch?v=txNvZ3Zndak>`__.
+
+* This is a great video explaining descriptive statistics and visualization, however there are commercials at various points.  Just ignore or skip ahead.
+
+.. list-table:: Commercials
+   :align: center
+   :header-rows: 1
+   :widths:  10 10 80
+   
+   * - **Start**
+     - **Stop**
+     - 
+   * - 3:44
+     - 4:08
+     - 
+   * - 6:16
+     - 7:26
+     - 
+   * - 14:00
+     - end
+     - 
```

Comment:
I would suggest removing this table and just leaving in the warning about the video having commercials. The commercial times are no longer accurate. That way no one has to update the page later if the times change again.